main.js

```diff
@@ -75,7 +75,7 @@ if (handleStartupEvent()) {
   console.log('OpenBazaar started on Windows...');
 }
 
-const serverPath = `${__dirname}${path.sep}..${path.sep}OpenBazaar-Server${path.sep}`;
+const serverPath = `${__dirname}${path.sep}..${path.sep}openbazaar-go${path.sep}`;
```

Comment:
> a.) That folder would also have to have the daemon binary but yes it's possible.

Oh right. Yeah, that makes it even less likelier

> b) I think that's a good option and much less brittle.

Ok, I create a bug #296. It's low priority for now. I don't think we'd need it until we have a use case for the client needing to know if it was started via the client-only package.